docusaurus/docs/create-a-plugin/develop-a-plugin/set-up-github-workflows.md

```diff
@@ -26,6 +26,40 @@ The CI (`ci.yml`) workflow is designed to lint, type check, and build the fronte
 
 This workflow requires a Grafana Cloud API key. Before you begin, follow the instructions for [generating an Access Policy token](../../publish-a-plugin/sign-a-plugin#generate-an-access-policy-token).
 
+You can store Access Policy token as a Repository Secret in GitHub:
+
+1. Access Repository Settings:
+
+- Go to your GitHub repository.
+- Navigate to the "Settings" tab.
+
+2. In the left sidebar, click on Secrets and Variables -> Actions
+3. Click on the "New repository secret" button.
+4. Add Secret Information:
+
+- Enter a descriptive name for your secret (e.g., GRAFANA_API_KEY).
```

Comment:
I was under the impression that we **require** a specific name? (I could be mistaken) And assume we'd want to reference access policy one rather than the deprecated one?

Reply:
You are right!  I have changed it using the name we expected